openapi/connector_mgmt-private-admin.yaml

```diff
@@ -1055,6 +1114,21 @@ components:
               type: string
               #format: date-time
 
+    ConnectorRequest:
+      required:
+        - desired_state
+      properties:
+        desired_state:
+          $ref: "#/components/schemas/ConnectorDesiredState"
```

Comment:
Reuse the type from the public API schema using `connector_mgmt.yaml#/components/schemas/ConnectorDesiredState`.

Reply:
fixed in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1291/commits/7ade896398fb40d4ffc9108ed504603badb87366